tests/suites/test_suite_bignum.misc.data

```diff
@@ -1788,6 +1788,51 @@ mpi_fill_random:16:15:0:MBEDTLS_ERR_ENTROPY_SOURCE_FAILED
 Fill random: MAX_SIZE bytes, RNG failure after MAX_SIZE-1 bytes
 mpi_fill_random:MBEDTLS_MPI_MAX_SIZE:MBEDTLS_MPI_MAX_SIZE-1:0:MBEDTLS_ERR_ENTROPY_SOURCE_FAILED
 
+MPI core random basic: 0..1
+mpi_core_random_basic:0:"01":0
+
+MPI core random basic: 0..2
+mpi_core_random_basic:0:"02":0
+
+MPI core random basic: 1..2
+mpi_core_random_basic:1:"02":0
+
+MPI core random basic: 2^30..2^31
+mpi_core_random_basic:0x40000000:"80000000":0
+
+MPI core random basic: 0..2^128
+mpi_core_random_basic:0x40000000:"0100000000000000000000000000000000":0
+
+MPI core random basic: 2^30..2^129
+mpi_core_random_basic:0x40000000:"0200000000000000000000000000000000":0
+
+# Use the same data values for mpi_core_random_basic->NOT_ACCEPTABLE
```

Comment:
There are a few pairs of test cases where we want to use the same values as test inputs. Not enough to really justify ensuring this via a script (but I could be convinced otherwise). They should be consecutive in the .data file. But quite possibly `mpi_core_random_basic` and `mpi_random_values` should be in different suites, so they couldn't have their test cases together. I don't like requesting a correspondence between different files.

If we [have a separate test suite for the random family of functions](https://github.com/Mbed-TLS/mbedtls/pull/6303/files#r1037632183), we can keep the pairs of test cases consecutive.